shared/rakefiles/entrypoint.rake

```diff
@@ -108,22 +108,30 @@ task :unlock => [:set_vars] do
     done'"
 end
 
-desc '[ADVANCED] Run arbitrary exekube command -- rake sh["kubectl --namespace gpii get pods"]'
-task :sh, [:cmd, :plain_shell] => [:set_vars] do |taskname, args|
+desc "[ADVANCED] Run arbitrary command in exekube container via rake wrapper (with secrets set) -- rake sh['kubectl exec --n gpii couchdb-couchdb-0 -c \
+couchdb -- curl -s http://$TF_VAR_secret_couchdb_admin_username:$TF_VAR_secret_couchdb_admin_password@127.0.0.1:5984/gpii/_all_docs']"
+task :sh, [:cmd] => [:set_vars] do |taskname, args|
   if args[:cmd]
     cmd = args[:cmd]
   else
     puts "Argument :cmd -- the command to run inside the exekube container -- not present, defaulting to bash"
     cmd = "bash"
   end
-  if args[:plain_shell]
-    sh "#{@exekube_cmd} #{cmd}"
+  sh "#{@exekube_cmd} rake xk['#{cmd}',skip_secret_mgmt,preserve_stderr]"
+end
+
+desc "[ADVANCED] Run arbitrary command in exekube container via plain shell -- rake plain_sh['kubectl --namespace gpii get pods']"
+task :plain_sh, [:cmd] => [:set_vars] do |taskname, args|
+  if args[:cmd]
+    cmd = args[:cmd]
   else
-    sh "#{@exekube_cmd} rake xk['#{cmd}',skip_secret_mgmt,preserve_stderr]"
+    puts "Argument :cmd -- the command to run inside the exekube container -- not present, defaulting to bash"
+    cmd = "vash"
```

Comment:
Unless [this guy](https://en.wikipedia.org/wiki/Vash_the_Stampede) is a shell now, this might be a typo. ;)